assets/scss/templates/_guidelines.scss

```diff
@@ -1,12 +1,12 @@
-main {
-  padding: 0 5%;
-}
+// main {
```

Comment:
Please remove commented code if not needed, or add a `TODO:` with a description, if it's something to think later.

Reply:
Se borra, no problem.